kb-s2i/README.md

```diff
@@ -0,0 +1,12 @@
+# kb-s2i
+
```

Comment:
Write what all this is about without requiring the reader to know intrinsics of OpenShift and containers. A paragraph or two is fine.

Reply:
"The files in this folder is used for automatic builds, tests and image creation using OpenShift at the Royal Danish Library"

Reply:
Taken in